pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/HadoopSegmentBuildPushOfflineClusterIntegrationTest.java

```diff
@@ -155,13 +155,13 @@ private void generateAndPushSegmentsFromHadoop()
     properties.setProperty(JobConfigConstants.PUSH_TO_PORT, getDefaultControllerConfiguration().getControllerPort());
 
     // Run segment creation job
-    SegmentCreationJob creationJob = new SegmentCreationJob("TestSegmentCreation", properties);
+    SegmentCreationJob creationJob = new SegmentCreationJob(properties);
```

Comment:
I think that it's better to expose `JobName` to be configurable. It's fine to include that config within `properties` and read it back along with other config instead of having it as an input variable for constructor.

Reply:
Job name is inferred in the job class. Other properties should be passed through `properties`.